pkg/services/ngalert/metrics/remote_alertmanager.go

```diff
@@ -13,72 +13,79 @@ const (
 )
 
 type RemoteAlertmanager struct {
-	Info                  *prometheus.GaugeVec
-	RequestLatency        *instrument.HistogramCollector
-	LastReadinessCheck    prometheus.Gauge
-	ConfigSyncsTotal      prometheus.Counter
+	ConfigSizeBytes       *prometheus.GaugeVec
```

Comment:
Adding `ConfigSizeBytes` is the only actual change. The first commit was just alphabetically ordering the existing metrics

Reply:
Ahh that makes sense. Ok, will pay attention to that